dev-haskell/flatparse/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>haskell@gentoo.org</email>
+		<name>Gentoo Haskell</name>
+	</maintainer>
+	<use>
+		<flag name="dump">dump core, stg and cmm to files</flag>
+		<flag name="llvm">use llvm for building</flag>
```

Comment:
- [`flatparse.cabal`](https://hackage.haskell.org/package/flatparse-0.5.2.1/flatparse.cabal)

    ```
      if flag(llvm)
        ghc-options: -fllvm
    ```

This is the only thing I'm halfway concerned with. It opens up the question of whether we should add this flag to everything via `haskell-cabal.eclass`, or drop the flag here and let people turn it on via `HCFLAGS` in `make.conf`.

This isn't a reason to block the PR, but I thought it was worth bringing up.